Gemfile

```diff
@@ -74,7 +74,7 @@ gem 'stringex', git: 'https://github.com/pulibrary/stringex.git', tag: 'vpton.2.
 gem 'mail_form'
 gem 'string_rtl'
 
-gem 'requests', git: 'https://github.com/pulibrary/requests.git', branch: 'alma'
+gem 'requests', git: 'https://github.com/pulibrary/requests.git', branch: 'main'
```

Comment:
I don't think we want to point back to main. we will lose all the requests updates

Reply:
That's strange, I fixed this last week, I am sorry, I will see how this regressed into my branch.

Reply:
the rebase fixed it.